activejob/test/cases/translation_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'helper'
+require 'jobs/translated_hello_job'
+
+class TranslationTest < ActiveSupport::TestCase
+  setup do
+    JobBuffer.clear
+    I18n.available_locales = [:en, :de]
+    @job = TranslatedHelloJob.new('Johannes')
+  end
+
+  teardown do
+    I18n.available_locales = [:en]
+  end
+
+  test 'it performs the job in the given locale' do
+    @job.locale = :de
+    @job.perform_now
```

Comment:
Sorry if I'm missing something but since the original issue was on `#perform_later`, there should certainly be a test for it, no ?

However with the following:

~~~ruby
test 'it performs the job in the given locale' do
  I18n.locale = :de
  TranslatedHelloJob.perform_later('Johannes')
  assert_equal "Johannes says Guten Tag", JobBuffer.last_value
end
~~~

the only failure is with Sucker Punch on my machine without your patch ; that seems strange.